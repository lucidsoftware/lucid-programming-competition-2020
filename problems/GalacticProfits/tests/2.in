6
1
Rocket_Builders
1 1 1 1 1 6
