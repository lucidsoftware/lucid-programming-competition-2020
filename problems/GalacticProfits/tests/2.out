Max: 125000
Min: 0
